app/src/main/res/values-fr/strings.xml

```diff
@@ -824,12 +824,12 @@ Basé sur le thème sombre Indigo"</string>
     <string name="downloading_reddit_video_finished">"Téléchargé"</string>
 
     <!-- Fuzzy -->
-    <string name="downloading_reddit_video_failed_cannot_get_cache_directory">"Échec du téléchargement: impossible d'accéder au répertoire de cache"</string>
-    <string name="downloading_reddit_video_failed_cannot_download_video">"Échec du téléchargement: impossible de télécharger la vidéo"</string>
-    <string name="downloading_reddit_video_failed_cannot_save_video">"Échec du téléchargement: impossible d'enregistrer la vidéo dans le répertoire de cache"</string>
-    <string name="downloading_reddit_video_failed_cannot_save_audio">"Échec du téléchargement: impossible d'enregistrer l'audio dans le répertoire de cache"</string>
-    <string name="downloading_reddit_video_failed_cannot_mux">"Échec du téléchargement: impossible de multiplexer la vidéo et l'audio"</string>
-    <string name="downloading_reddit_video_failed_cannot_save_mux_video">"Échec du téléchargement: impossible d'enregistrer la vidéo dans le répertoire public"</string>
+    <string name="downloading_reddit_video_failed_cannot_get_cache_directory">"Échec du téléchargement : impossible d'accéder au répertoire de cache"</string>
```

Comment:
Hi! I am not familiar with French. But why is a space necessary here?

Reply:
That's our punctation rules ;)
Like in English, you would write "What?", we would write " What ?". _(Well, « Quoi ? »...)_